cicd/builds/running-entitled-builds.adoc

```diff
@@ -22,6 +22,10 @@ include::modules/builds-source-input-satellite-config.adoc[leveloffset=+2]
 
 include::modules/builds-strategy-docker-entitled-satellite.adoc[leveloffset=+2]
 
+.Additional resources
```

Comment:
Suggestion: 

- Please add [role="_abstract"] before L9.
- I see that the [OCP guidelines for additional resources](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections) mention this: "the . formatting of the include statement indicates that the resource applies specifically to the module and not to the assembly. For example:

[role="_additional-resources"]
.Additional resources
* link:some-url.com[Human readable label]
* xref:some_xref[Human readable label]
* xref:some_other_xref[Human readable label]"

 Based on this guideline, I believe you should add _[role="_additional-resources"]_ before L25. PTAL and do the needful.

Reply:
That's correct. Updated. Thank you.